README.md

```diff
@@ -128,6 +90,17 @@ implementation from Agenzia per l'Italia Digitale.
 
         $ ./gencert-private.sh
 
+## Validate generated certificate (with Docker)
+
+  if you want to check the generated certificate file (default: `generated-certs/crt.pem`)
+  you need only to launch the script `validate-crt-with-docker.sh` and check the output in the console.
+
+  if you want to change the certificate file to validate, just change the env variable `CERT_FILE` inside
+  `validate-crt-with-docker.sh` and launch the script.
+
+  NOTE: the first execution takes some time, because the docker image needs to compile
+  the python package `cryptography` from source; after that, every other execution is immediate.
+
```

Comment:
This section should be revised according to the changes that will occur on `validate-crt-with-docker.sh`

Reply:
done! 👍🏻
@psmiraglia please resolve the conversation if the edits are ok!